src/draggable.jsx

```diff
@@ -3,29 +3,51 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import DomDraggable from '@telerik/kendo-draggable';
 
+function noop() { }
+
 export default function(Component) {
     return class extends React.Component {
+        constructor(props) {
+            super(props);
+            const { onPress = noop, onDrag = noop, onRelease = noop } = props;
+            this.onPressHandler = onPress;
+            this.onDragHandler = onDrag;
+            this.onReleaseHandler = onRelease;
+        }
+
```

Comment:
I don't like this part. Is there a better, more concise way to deal with it // @rkonstantinov @rusev